platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/PreConsensusEventHandler.java

```diff
@@ -51,7 +50,7 @@
  * SwirldState implemented). It contains a thread queue that contains a queue of pre-consensus events (q1) and a
  * SwirldStateManager which applies those events to the state
  */
-public class PreConsensusEventHandler implements PreConsensusEventObserver, Clearable, Startable {
+public class PreConsensusEventHandler implements Clearable, Startable {
```

Comment:
While you're in this class, the class javadoc could be cleaned up.

Reply:
Fixed